include/uuid/uuid_gen.h

```diff
@@ -48,6 +48,7 @@ typedef const struct _st_uuid* cpst_uuid_v6;
 
 typedef struct _st_uuid_str st_uuid_v6_str, *pst_uuid_v6_str;
 typedef const struct _st_uuid_str* cpst_uuid_v6_str;
+typedef struct _st_uuid st_uuid_v6, *UUID;
```

Comment:
Can we start using these guidelines:

https://confluence.ultracruise.gm.com/pages/viewpage.action?pageId=153357544#IVStreamerC++CodingStyleStandard-Typedef

Reply:
this module should be refactored in any case